src/main/java/com/rqlite/impl/NodeUnavailableException.java

```diff
@@ -0,0 +1,10 @@
+package com.rqlite.impl;
+
+/**
+ * This exception is thrown when rqlite-java cannot connect to a rqlite node.
+ **/
+public class NodeUnavailableException extends Exception {
```

Comment:
See comment above. I think this should be defined outside of the implementation, in the same place the interface is defined.